internal/server/data/data_test.go

```diff
@@ -213,3 +214,37 @@ func TestDB_Begin(t *testing.T) {
 		})
 	})
 }
+
+func TestLongRunningQueriesAreCancelled(t *testing.T) {
+	runDBTests(t, func(t *testing.T, db *DB) {
```

Comment:
Since we're sleeping in this test, I think we need to add the following to the top of the test to allow it to be skipped

```go
if testing.Short() {
    t.Skip("too slow for short run")
}
```